templates/services/backend/manifest.yml

```diff
@@ -1,15 +1,15 @@
-# The manifest for the "{{.Name}}" application.
+# The manifest for the "{{.Name}}" service.
 # Read the full specification for the "{{.Type}}" type at:
-#  https://github.com/aws/amazon-ecs-cli-v2/wiki/Manifests#backend-app
+#  https://github.com/aws/amazon-ecs-cli-v2/wiki/Manifests#backend-svc
 
-# Your application name will be used in naming your resources like log groups, services, etc.
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
 name: {{.Name}}
 
-# Your application is reachable at "http://{{.Name}}.${ECS_APP_DISCOVERY_ENDPOINT}:{{.Image.Port}}" but is not public.
+# Your service is reachable at "http://{{.Name}}.${ECS_SVC_DISCOVERY_ENDPOINT}:{{.Image.Port}}" but is not public.
```

Comment:
nice, I like this a lot!
This means that we need to update the cloudformation stack with this new environment variable right?
https://github.com/aws/amazon-ecs-cli-v2/blob/da87752a68a64542b8612d3ec9047fc17be9018a/templates/services/common/cf/envvars.yml#L7-L8

Would you mind doing that part of this PR? I am worried that we will forget to update it when we get to the `stack` pkg

Reply:
yeah sure! Good call